lib/error-matcher.js

```diff
@@ -77,12 +90,22 @@ module.exports = (function () {
   };
 
   ErrorMatcher.prototype.set = function (regex, cwd, output) {
-    try {
-      this.regex = XRegExp(regex);
-    } catch (err) {
-      this.regex = null;
-      return this.emit('error', 'Error parsing regex:\n' + err);
+    this.regex = [];
+    for (var i in regex) {
+      var curRegex = null;
+      try {
+        curRegex = XRegExp(regex[i]);
+      } catch (err) {
+        this.emit('error', 'Error parsing regex:\n' + err);
+      }
+      if (curRegex !== null) {
+        this.regex.push(curRegex);
+      }
```

Comment:
Hmm, I think this can be better done with `map`. Something like this (untested):

```javascript
this.regex = _.map(regex, function (r) {
  try {
    this.regex.push(XRegExp(r));
  } catch (err) {
    this.emit('error', 'Error parsing regex: ' + err);
  }
});
```